apps/aecore/lib/aecore/util/header.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Aecore.Util.Header do
+  @moduledoc """
+  Header utilities
+  """
+  alias Aecore.Chain.KeyHeader
+  alias Aecore.Chain.MicroHeader
+
+  @spec hash(KeyHeader.t() | MicroHeader.t()) :: binary()
+  def hash(header) do
+    case header do
+      %KeyHeader{} ->
```

Comment:
Code reduction

Reply:
This change is present in master-ng, I forgot to set that as the merge target